web/src/app/util/RequireConfig.js

```diff
@@ -71,6 +71,32 @@ function isTrue(value) {
   return Boolean(value)
 }
 
+const mapConfig = value =>
+  _.chain(value.config)
+    .groupBy('id')
+    .mapValues(v => parseValue(v[0].type, v[0].value))
+    .value()
+
+// useSessionInfo returns an object with `isAdmin` true if the current
+// session is an admin, and a `userID` value set to the current users ID.
+export function useSessionInfo() {
```

Comment:
would `getSessionInfo`/config make more sense here? i was a little confused as to what it was doing until i got to the file and was able to read through it

Reply:
Since it is a hook they recommend to always start with a `use` prefix to separate it from non-hooks functions, even if `get` makes more sense. :)